lib/rspec/core/configuration.rb

```diff
@@ -778,7 +780,7 @@ def full_backtrace=(true_or_false)
         @backtrace_formatter.full_backtrace = true_or_false
       end
 
-      # Returns the configuration option for color, but should not
+      # Deprecated: returns the configuration option for color, but should not
```

Comment:
Also, it'd be nice if the docs explained why it was deprecated and what the user's options are (e.g. set nothing or `--force-color` instead).